File: 2nd Year/2nd Semester/MPP/PROIECT-C#/Lab1/TriatlonNetworking/dto/ProbaDTO.cs
using System;

namespace TriatlonNetworking.dto
{
    [Serializable]
    public class ProbaDTO
    {
        public int? Id { get; set; }
        public TipProbaDTO TipProba { get; set; }
        public ArbitruDTO Arbitru { get; set; }

        public ProbaDTO(int? id, TipProbaDTO tipProba, ArbitruDTO arbitru)
        {
            Id = id;
            TipProba = tipProba;
            Arbitru = arbitru;
        }
    }
}

File: 2nd Year/2nd Semester/MPP/Triatlon-Java/TriatlonNetworking/src/main/java/triatlon/network/dto/ProbaDTO.java
package triatlon.network.dto;

import java.io.Serializable;

public class ProbaDTO implements Serializable {
    private Integer id;
    private TipProbaDTO tipProba;
    private ArbitruDTO arbitru;

    public ProbaDTO(Integer id, TipProbaDTO tipProba, ArbitruDTO arbitru) {
        this.id = id;
        this.tipProba = tipProba;
        this.arbitru = arbitru;
    }

    public Integer getId() {
        return id;
    }

    public TipProbaDTO getTipProba() {
        return tipProba;
    }

    public ArbitruDTO getArbitru() {
        return arbitru;
    }

    public void setTipProba(TipProbaDTO tipProba) {
        this.tipProba = tipProba;
    }
}